styles/style-main.css

```diff
@@ -305,6 +412,81 @@ header {
   background-color: #febf52;
 }
 
+.dots-tile {
+  display: none;
+}
+
+@media only screen and (min-width: 992px) {
+  .about {
+    flex-direction: row-reverse;
+    justify-content: center;
+    margin: 160px;
+  }
+
+  .about .right {
+    display: flex;
+    flex-direction: column;
+    padding-left: 80px;
+  }
+
+  .about h1 {
+    font-size: 110px;
+  }
+
+  .about p {
+    margin-top: 16px;
+  }
+
+  #get-my-cv {
+    font-size: 18px;
+    align-self: flex-start;
+    width: 254px;
+  }
+
+  .knowledge {
+    display: grid;
+    grid-template-columns: 289px 289px;
+    grid-template-rows: 289px 289px;
+    margin-top: 0;
+  }
+
+  .knowledge h2 {
+    font-size: 20px;
+    font-weight: 800;
+  }
+
+  .knowledge ul {
+    font-size: 18px;
+    font-weight: 600;
+  }
+
+  .languages {
+    grid-column: 1/2;
+    margin: 0;
+  }
+
+  .frameworks {
+    grid-row: 1/2;
+    grid-column: 2/3;
+    margin: 0;
+  }
+
+  .skills {
+    grid-row: 2/3;
+    grid-column: 1/2;
+    margin: 0;
+  }
+
+  .dots-tile {
+    display: initial;
+    grid-row: 2/3;
+    grid-column: 2/3;
+    background-image: url(../images/image_dots_with_teal.svg);
+    background-repeat: no-repeat;
+    background-position: right bottom;
+  }
+}
```

Comment:
- [ ] I know you have worked hard on styling this project :clap:  but one of the requirements says that you should stick to the design as much as possible. Kindly work on it and your project will be okay.  check the difference between your design and the original template :point_down:  then go ahead and improve yours.

|  |Your design  |    |
|- |---------------- | - |
| |![raw githack com_eduardosancho_setup_and_mobile_version_skeleton_develop-desktop-version_index html](https://user-images.githubusercontent.com/65068771/144493474-f711fb48-e0a8-41d0-a52d-a5b93ea73c0b.png) | |

|  | The template  |    |
|- |---------------- | - |
|  | ![Template 8-main](https://user-images.githubusercontent.com/65068771/144493523-8b3117bd-721a-4898-ae92-f02aa8a5329f.png) |  |

- [ ] Kindly follow font, colors, images,  margins and positions  of element in the original template.

### I know that this will take time to implement but consider it as part of your training thanks.